crates/rust-client/src/rpc/tonic_client/mod.rs

```diff
@@ -223,16 +223,16 @@ impl NodeRpcClient for TonicRpcClient {
         response.into_inner().try_into()
     }
 
-    /// Sends a `GetAccountDetailsRequest` to the Miden node, and extracts an [FetchedAccount] from
-    /// the `GetAccountDetailsResponse` response.
+    /// Sends a `GetAccountDetailsRequest` to the Miden node, and extracts an [`FetchedAccount`]
+    /// from the `GetAccountDetailsResponse` response.
     ///
     /// # Errors
     ///
     /// This function will return an error if:
     ///
     /// - There was an error sending the request to the node.
     /// - The answer had a `None` for one of the expected fields (account, summary,
-    ///   account_commitment, details).
+    ///   `account_commitment`, details).
```

Comment:
nit: should all these be wrapped by ` ? Because they all reference specific fields:
```suggestion
   /// - The answer had a `None` for one of the expected fields (`account`, `summary`,
   ///   `account_commitment`, `details`).
```